content/docs/2.15/deploy.md

```diff
@@ -4,9 +4,9 @@ title = "Deploying KEDA"
 
 We provide a few approaches to deploy KEDA runtime in your Kubernetes clusters:
 
-- [Helm charts](#helm)
-- [Operator Hub](#operatorhub)
-- [YAML declarations](#yaml)
```

Comment:
I'm not sure where these should link, so i've just removed the links

Reply:
They're explicitly tagged anchors internal to this file. For example, line 15:
```
## Deploying with Helm {#helm}
```

They work in the existing documentation. If they don't break anything I'd leave them in.

Reply:
Update: As you point out, they might be what's causing the reference problem, so let's remove them and see what happens.